src/components/paymentForm/PaymentForm.scss

```diff
@@ -0,0 +1,6 @@
+.gray-text {
+  color: #3a3a3a;
```

Comment:
made a variable for each color and it's better if you asked your teammates about what color did they use and use the same
maybe you code put all the css global variable somewhere shared and used it from there so no code will be wrote twice